app/oracle.py

```diff
@@ -152,6 +152,16 @@
 abi = json.loads(a)
 oracle = w3.eth.contract(abi=abi['abi'], address=oracle_address)
 
+quorum_size = oracle.functions.getQuorum().call()
+oracle_members = oracle.functions.getOracleMembers().call()
+
+if not dry_run and account:
+    try:
+        oracle_holder_index = oracle_members.index(account.address)
+    except ValueError:
+        oracle_holder_index = 0
+        logger.warning('Account is not oracle member.')
```

Comment:
I think it's better to get the quorum and member index just before the report. This would only happen once a day, but we wouldn't need to ask oracle owners to restart in case of any parameter changes in the contract